### PR TITLE
Export qti api

### PR DESCRIPTION
**Pull request by chrisphpgarcia**
required tao-core origin/mime-type-accept

### CODE REVIEW

File: controller/RestQtiItem.php [hunk 1]
```diff
@@ -161,6 +162,53 @@ public function createQtiItem()
...
     }
+    
+    /**
+     * render item rdf xml
+     * @author christophe GARCIA <christopheg@taotesting.com>
+     */
+    public function export() {
+        try {
+            if ($this->getRequestMethod()!=Request::HTTP_GET) {
+                    throw new \common_exception_NotImplemented('Only GET method is accepted to create empty item.');
```
Comment (jbout) [inline controller/RestQtiItem.php]: Copy/past message

File: controller/RestQtiItem.php [hunk 2]
```diff
@@ -161,6 +162,53 @@ public function createQtiItem()
...
+            }
+            
+            if(!$this->hasRequestParameter('id')) {
+                return $this->returnFailure(new common_exception_MissingParameter('required parameter `id` is missing'));
+             
+            } 
+            
+            $id = $this->getRequestParameter('id');
+            $class = 'http://www.tao.lu/Ontologies/TAOItem.rdf#Item';
```
Comment (jbout) [inline controller/RestQtiItem.php]: unused variable, pls use constants for RDF classes

Comment (chrisphpgarcia) [inline controller/RestQtiItem.php]: suppress

File: controller/RestQtiItem.php (hunk 1 above)
Comment (chrisphpgarcia) [inline controller/RestQtiItem.php]: done

**Comment by jbout** [comment]
Requires: https://github.com/oat-sa/tao-core/pull/866

**Pull request closed without merging**